src/uu/env/src/env.rs

```diff
@@ -602,6 +655,44 @@ fn apply_specified_env_vars(opts: &Options<'_>) {
     }
 }
 
+fn apply_ignore_signal(opts: &Options<'_>) -> UResult<()> {
+    for sig in &opts.ignore_signal {
+        let sig_str = match sig.to_str() {
+            Some(s) => s,
+            None => {
+                return Err(USimpleError::new(
+                    1,
+                    format!("invalid signal name: {}", sig.quote()),
+                ))
+            }
+        };
+        let sig_value = parse_signal_value(sig_str)?;
+        let sig: Option<Signal> = if sig_str == "STOP" {
+            None
+        } else {
+            let sig = (sig_value as i32)
+                .try_into()
+                .map_err(|e| std::io::Error::from_raw_os_error(e as i32))?;
+            Some(sig)
+        };
+
+        match sig {
+            Some(signal) => ignore_signal(signal),
+            None => {
+                return Err(USimpleError::new(
+                    125,
+                    "cannot ignore STOP signal".to_string(),
+                ))
+            }
+        }
+    }
+    Ok(())
+}
+
+fn ignore_signal(sig: Signal) {
+    let _ = unsafe { signal(sig, SigIgn) }.map(|_| ());
```

Comment:
Two problems:
- Please add a comment saying `// SAFETY: ` followed by a reasoning why this is probably hopefully reasonably safe. (I agree that it is safe, but it should be documented.)
- Ignoring the returned `Result` is a big no-no. This can and will fail, for various reasons. You probably already discovered that ignoring SIGSTOP doesn't work. This is the place where it should be detected.